src/api/tests/test_feishu.py

```
@@ -1,5 +1,77 @@
+    shifu_info = SimpleNamespace(title="Test Course")
+
+    monkeypatch.setattr(order_funs, "query_buy_record", lambda _app, _id: order_info)
+    monkeypatch.setattr(order_funs, "load_user_aggregate", lambda _id: aggregate)
+    monkeypatch.setattr(order_funs, "get_shifu_info", lambda _app, _cid, _p: shifu_info)
+
+    class FakeQuery:
+        def __init__(self, first_value=None, count_value=0):
+            self._first_value = first_value
+            self._count_value = count_value
+
+        def filter(self, *args, **kwargs):
```

Ruff (0.14.13)

Unused method argument: `args`

(ARG002)